ci/helm/values/values-nuxeo.yaml.gotmpl

```diff
@@ -0,0 +1,36 @@
+{{- $localDomain := printf "%s.svc.cluster.local" (requiredEnv "NAMESPACE") -}}
+image:
+  repository: {{ printf "%s/nuxeo/nuxeo-compound-documents-ftests" (requiredEnv "DOCKER_REGISTRY") }}
+  tag: {{ requiredEnv "VERSION" | quote }}
+  pullSecrets:
+  - name: kubernetes-docker-cfg
+podLabels:
+  {{ tpl (readFile "labels.yaml.gotmpl") . | nindent 2 }}
+  dev.nuxeo.com/app: nuxeo
+  dev.nuxeo.com/resource: pod
+service:
+  labels:
+    {{ tpl (readFile "labels.yaml.gotmpl") . | nindent 4 }}
+    dev.nuxeo.com/app: nuxeo
+    dev.nuxeo.com/resource: service
+mongodb:
+  enabled: true
+  host: {{ printf "mongodb.%s" $localDomain }}
+packages: nuxeo-opensearch1-embed nuxeo-search-client-opensearch1 nuxeo-web-ui /packages/nuxeo-compound-documents-package-*.zip
```

Comment:
What not deploying the OpenSearch that is present in the helmfile files?

Reply:
Why not :) Keep in mind this commit is temporary and to be removed once we align on `2025.0.129` and greater. See https://hyland.atlassian.net/browse/NXP-33041